packages/api/ormconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I think we'll probably need to avoid committing this file and providing the configuration inline during our connection setup code since we're going to dynamically pull the host, port, username, database, and password.

Reply:
Sure, using this to get the models together.  Will remove if not needed.

Reply:
Removed in 5852a8bfc32a595be3f925a56a516c859e56e37a